sqlserver/datadog_checks/sqlserver/const.py

```diff
@@ -9,6 +9,22 @@
 PERF_COUNTER_BULK_COUNT = 272696576
 PERF_COUNTER_LARGE_RAWCOUNT = 65792
 
+# Engine Editions; see:
+# https://docs.microsoft.com/en-us/sql/t-sql/functions/serverproperty-transact-sql
+ENGINE_EDITION_PERSONAL = 1
```

Comment:
Could this be a mapping instead of a lot of constants? Also, it seems like most of these aren't used now, do you think they will be used in the future?

Reply:
What would be the purpose of a mapping?

I'm not sure which of them will be used in the future, but as we add more functionality which is only supported in certain editions, they will be used in those cases to allowlist/blocklist. Since the values are extremely unlikely to change, I added them all upfront.

Reply:
Sure, that makes sense! I was envisioning a mapping could potentially be more readable but I think given the way these are used now it would not be beneficial.